src/components/HTMLEngineProvider/htmlEngineUtils.js

```diff
@@ -52,7 +54,7 @@ function isChildOfNode(tnode, predicate) {
  * @returns {Boolean}
  */
 function isChildOfComment(tnode) {
-    return isChildOfNode(tnode, (node) => isCommentTag(node.domNode.name));
+    return isChildOfNode(tnode, (node) => isCommentTag(lodashGet(node, 'domNode.name', '')));
```

Comment:
Also adding this change since the `isCommentTag` expects a `string` and we may pass `undefined` here without the default